cli/src/main.rs

```diff
@@ -185,9 +185,14 @@ fn main() {
                     let disk_path =
                         dev.options.get("disk").unwrap().as_str().unwrap();
 
+                    let readonly: bool = || -> Option<bool> {
+                        dev.options.get("readonly")?.as_str()?.parse().ok()
+                    }()
+                    .unwrap_or(false);
```

Comment:
As we get further into the config stuff, I imagine we'll want some sort of convenience access function for expected-boolean data.

Reply:
Agreed - there's some tension right now between "settings that we would only ever do on local objects" vs "settings that will actually be applicable once we get e.g. cloud storage, remote images up".

100% on board with abstracting away the parsing though.